### PR TITLE
Minimum delegation

### PR DESCRIPTION
**Pull request by jtakalai**

### CODE REVIEW

File: packages/network-contracts/contracts/OperatorTokenomics/Operator.sol [hunk 1]
```diff
@@ -181,6 +181,11 @@ contract Operator is Initializable, ERC2771ContextUpgradeable, IERC677Receiver,
         return super._msgData();
     }
 
+    function _transfer(address from, address to, uint amount) internal override {
+        require(balanceOf(from) >= amount + minimumDelegationWei, "error_minDelegationNotReached");
```
Comment (jtakalai) [inline packages/network-contracts/contracts/OperatorTokenomics/Operator.sol:185]: also allow zero like discussed? Maybe `error_delegationAmountBelowMinimum`.

Comment (jtakalai) [reply packages/network-contracts/contracts/OperatorTokenomics/Operator.sol:185]: I think also add a test where we notice that delegation goes to zero? Could be difficult without explicit event.

Maybe add event `BalanceUpdate(uint)` that sends the current balance to subgraph? Could be better than tracking differences. That would need to be added to other places too where pool token balances change.

I created a separate ticket for that: ETH-521

Comment (samigabor) [reply packages/network-contracts/contracts/OperatorTokenomics/Operator.sol]: Enabled transferring the full amount (e.g. fully undelegate). 
Handled the BalanceUpdate event in eth-521
